src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -179,9 +185,11 @@ private void updateFilteredItems() {
     boolean showPasswordProtectedGames = showPasswordProtectedGamesCheckBox.isSelected();
     boolean showModdedGames = showModdedGamesCheckBox.isSelected();
 
-    filteredItems.setPredicate(OPEN_CUSTOM_GAMES_PREDICATE.and(gameInfoBean ->
-        (showPasswordProtectedGames || !gameInfoBean.isPasswordProtected())
-            && (showModdedGames || gameInfoBean.getSimMods().isEmpty())));
+    filteredItems.setPredicate(OPEN_CUSTOM_GAMES_PREDICATE
+        .and(gameInfoBean ->
+            (showPasswordProtectedGames || !gameInfoBean.isPasswordProtected())
+                && (showModdedGames || gameInfoBean.getSimMods().isEmpty()))
+        .and(rangePredicate));
```

Comment:
No filtering needs to be done !!! The server already filters the game so they fit your rating. If the rating it not enforced the games are not filtered by the server and the rating only acts as a description.